README.md

```diff
@@ -71,7 +71,7 @@ your config/application.rb to apply RuboCop autocorrection to code generated by
 module YourCoolApp
   class Application < Rails::Application
     config.generators.after_generate do |files|
-      system('bundle exec rubocop -A ' + files.shelljoin, exception: true)
+      system('bundle exec rubocop --fail-level=F -A ' + files.shelljoin, exception: true)
```

Comment:
I'm wondering if `--fail-level=E` should be specified instead of `--fail-level=F`.

Reply:
And that's just my two cents. Can you specify `-A` option before `--fail-level` option to respect the context?